### ISSUE TITLE
calling write fails with E17 path is a directory

### ISSUE THREAD

**Issue by matu3ba** [elihunter173/dirbuf.nvim#50]
config:
```lua
use { 'elihunter173/dirbuf.nvim', config = function() require("dirbuf").setup { write_cmd = "DirbufSync -confirm" } end, }
```
However, manually running the command `:DirbufSync -confirm` works as expected.

The behavior occurs both from opening a directory and opening a file first and then calling `:Dirfuf %`
Is this behavior expected? If yes, then it should be documented.

Side question: Is there a difference in using `:e pathtodir` and `:Dirbuf pathtodir`?

**Comment by matu3ba** [comment]
known workaround: Use the first time `:w!` to force the write.
Subsequent writes will just work.

**Comment by ojroques** [comment]
I have the same issue on Neovim nightly. Maybe the same issue as #45.

**Comment by elihunter173** [comment]
> Is this behavior expected? If yes, then it should be documented.

This behavior is not expected. Are you using Neovim nightly? If yes, then I'll close this issue as a duplicate of #45 since I just re-opened that for investigation.

**Comment by matu3ba** [comment]
Yes, I am using Neovim nightly. Closed as duplicate of #45.
